ydb/core/mind/hive/hive_schema.h

```diff
@@ -298,6 +298,15 @@ struct Schema : NIceDb::Schema {
         using TColumns = TableColumns<Node, TabletType, MaxCount>;
     };
 
+    struct OperationsLog : Table<21> {
+        struct Timestamp : Column<1, NScheme::NTypeIds::Uint64> {};
+        struct User : Column<2, NScheme::NTypeIds::String> {};
+        struct Operation : Column<3, NScheme::NTypeIds::String> { using Type = NKikimrHive::TOperation; };
```

Comment:
I don't think you actually need these complex protobuf specs just for logging purposes. And think about migrations and rollbacks - it could became a huge pain...
I think just a free text would be more than enough. (or json if you need some structure to parse)